subsys/bluetooth/host/gatt.c

```diff
@@ -5696,3 +5697,153 @@ void bt_gatt_disconnected(struct bt_conn *conn)
 	remove_cf_cfg(conn);
 #endif
 }
+
+struct svc_uuid_data_state {
+	struct bt_data *data;
+	uint8_t *data_buf;
+	const size_t data_buf_size;
+	bool truncated;
+};
+
+/* This is an array of UUID values that we return via the
```

Comment:
I think this should be "... that we don't return ...", right?

We should also add info about the selection criteria for the entries of this list. I suggest "List of Service UUIDs that are mandatory on a GATT server. Adverts for these Service UUIDs are redundant. bt_gatt_get_svc_uuid_data() filters out entries that are on this list."